refocus-bdk-client.js

```diff
@@ -21,6 +21,7 @@ const moment = require('moment');
 const url = require('url');
 const request = require('superagent');
 const serialize = require('serialize-javascript');
+const { botName } = require('../../../config');
```

Comment:
personally i really dont like this solution of reaching outside of the bdk and relying on a certain file structure. I also can see in some bots botname isnt even in config. 

Would rather see us pass in botname in the same way we pass in the dev config but it would require updates to all bots. i think atleast this needs to point to package.json .name as we know that should always have a name 

@LongweiDeng @cassiodias not sure about your guys opinions on this

Reply:
good point, I agree

Reply:
I've changed it so that the botName is now passed into the bdk with the config when it is initalized. Is this preferable or is there a better way?

Reply:
That's a valid point, I agree.